Leetcode/24/Swap_Nodes_In_Pairs.cpp

```diff
@@ -0,0 +1,100 @@
+/*
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+  Problem Number in Leetcode: 24
+  Problem Link: https://leetcode.com/problems/swap-nodes-in-pairs/
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+*/
+
+#include <iostream>
+
+struct ListNode {
+      int val;
+      ListNode *next;
+      ListNode() : val(0), next(nullptr) {}
+      ListNode(int x) : val(x), next(nullptr) {}
+      ListNode(int x, ListNode *next) : val(x), next(next) {}
+};
+
+class Solution {
+public:
+    ListNode* swapPairs(ListNode* head) {
+        if(head == NULL || head->next == NULL) return head;
+          
+        // Initialize pointers
+        ListNode *newHead = head->next;
+        ListNode *tmp = head;
+        ListNode *prev = NULL;
+      
+        while(tmp != NULL && tmp->next != NULL){
```

Comment:
tmp && tmp-> next would help here. 

We can rename the variables for better clarity and use 'current' and 'previous' for 'tmp' and 'prev', respectively.